### ISSUE TITLE
[Enhancement Idea] Toggle On/Off

### ISSUE THREAD

**Issue by Enteleform** [jancborchardt/hyperminimal#18]
A toggle feature would make it much easier to reposition Hyper on Windows. The `alt + space + m` workaround is a bit cumbersome & doesn't work very well if the window is maximized.

**Comment by jancborchardt** [comment]
Ok, but then how would you toggle? Probably using a shortcut or menu entry, and then move. And isn't that more cumbersome than the key combo?

**Comment by Enteleform** [comment]
A key shortcut would be the most efficient way I think. 

Toggling the menu on & repositioning with the mouse is much faster than moving with the keyboard arrows IMO.

**Comment by jancborchardt** [comment]
So, kind of a key shortcut for the _key shortcut_ `alt + space + m`? ;) I’m not on Windows so I’m not having the issue – if you have a good proposal for a fitting key shortcut or even want to open a pull request implementing that, you are very welcome. :)

**Comment by Enteleform** [comment]
> So, kind of a key shortcut for the key shortcut alt + space + m?

Not really. That shortcut toggles into a window moving state that is affected by the keyboard, but not the mouse. I meant more of like disabling/enabling the CSS so that the title bar can be used to move the window, which would allow faster movement & window snapping.

**Comment by jancborchardt** [comment]
That's also an interesting proposal. Help and pull requests on this are very welcome.

Relevant here is also #19 which adds another solution to the Readme.
